File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/RefreshTokenResponse.java
package com.lattis.ellipse.data.network.model.response;

import com.google.gson.annotations.SerializedName;

public class RefreshTokenResponse extends AbstractApiResponse {

    @SerializedName("payload")
    public TokenResponse tokenResponse;

    public TokenResponse getTokenResponse() {
        return tokenResponse;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/authentication/RefreshTokenResponse.kt
package com.lattis.data.entity.response.authentication

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class RefreshTokenResponse : AbstractApiResponse() {

    @SerializedName("payload")
    var tokenResponse: TokenResponse? = null
}
